src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -7,40 +7,19 @@ const px10asRem = pxToRem(10)
 
 const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageVariables> = {
   root: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    display: 'inline-flex',
     position: 'relative',
-    marginTop: '1rem',
-    marginBottom: '1rem',
-    ...(p.mine && {
-      float: 'right',
-    }),
+    padding: pxToRem(14),
+    borderRadius: '0.3rem',
+    color: 'rgb(64, 64, 64)',
+    backgroundColor: p.mine ? v.messageColorMine : v.messageColor,
```

Comment:
I would add at least the color, borderRadius and the padding in the variables too.

Reply:
done